tools_docker/README.md

```diff
@@ -146,13 +145,61 @@ Exemple to use Tracim with ElasticSearch-ingest: (you need to create your elasti
 
 ⚠ After execute one of these command, Tracim will be reachable on your system on port 8080.
 
+#### Running with gocryptfs encryption
+
+This need the new specific Debian_New_Uwsgi docker (see build section). This is an experimental
+docker image, the new feature from this docker will maybe be merged to the standard docker or removed.
+Exemple with basic instance of Tracim (local usage with webdav and caldav) with encrypted storage:
+
+for this exemple,
+you first need to write password you want in `~/tracim/secret/password.txt` file.
+
+```bash
+echo "password" -> ~/tracim/secret/password.txt
```

Comment:
```suggestion
echo "password" > ~/tracim/secret/password.txt
```

Warn that it writes the password in the bash history and that people should edit the password with a text editor instead